class-author-profile-blocks.php

```
@@ -82,7 +84,86 @@ private function __construct() {
+		$this->register_blocks();
+	}
+
+	/**
+	 * Register admin components.
+	 *
+	 * @return void
+	 */
+	private function register_admin(): void {
+		new Admin();
+		new PluginLinks();
+	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'register_admin'. (undefined)

(UnusedPrivateMethod)